From ddbda328b37eb4b5f43fbd1dd8248c301fd2b30e Mon Sep 17 00:00:00 2001
From: Kamil Dudka <kdudka@redhat.com>
Date: Fri, 12 Apr 2013 14:13:42 +0200
Subject: [PATCH] tests: prevent test206, test1060, and test1061 from failing

... in case runtests.pl is invoked with non-default -b option

Fixes a regression caused by 1e29d275c643ef6aab7948f0f55a7a9397e56b42.
---
 tests/data/test1060 | 14 +++++++-------
 tests/data/test1061 | 14 +++++++-------
 tests/data/test206  | 14 +++++++-------
 3 files changed, 21 insertions(+), 21 deletions(-)

diff --git a/tests/data/test1060 b/tests/data/test1060
index da1be78..e303a89 100644
--- a/tests/data/test1060
+++ b/tests/data/test1060
@@ -874,7 +874,7 @@ crypto
 HTTP proxy CONNECT auth Digest, large headers and data
  </name>
  <command>
-http://test.remote.haxx.se.1060:%HTTPPORT/path/10600002 --proxy http://%HOSTIP:%HTTPPORT --proxy-user silly:person --proxy-digest --proxytunnel
+http://test.remote.haxx.se.1060:8990/path/10600002 --proxy http://%HOSTIP:%HTTPPORT --proxy-user silly:person --proxy-digest --proxytunnel
 </command>
 </client>
 
@@ -884,17 +884,17 @@ http://test.remote.haxx.se.1060:%HTTPPORT/path/10600002 --proxy http://%HOSTIP:%
 ^User-Agent: curl/.*
 </strip>
 <protocol>
-CONNECT test.remote.haxx.se.1060:%HTTPPORT HTTP/1.1
-Host: test.remote.haxx.se.1060:%HTTPPORT
+CONNECT test.remote.haxx.se.1060:8990 HTTP/1.1
+Host: test.remote.haxx.se.1060:8990
 Proxy-Connection: Keep-Alive
 
-CONNECT test.remote.haxx.se.1060:%HTTPPORT HTTP/1.1
-Host: test.remote.haxx.se.1060:%HTTPPORT
-Proxy-Authorization: Digest username="silly", realm="weirdorealm", nonce="12345", uri="test.remote.haxx.se.1060:%HTTPPORT", response="e1fbed39c26f4efe284adc0e576ff638"
+CONNECT test.remote.haxx.se.1060:8990 HTTP/1.1
+Host: test.remote.haxx.se.1060:8990
+Proxy-Authorization: Digest username="silly", realm="weirdorealm", nonce="12345", uri="test.remote.haxx.se.1060:8990", response="e1fbed39c26f4efe284adc0e576ff638"
 Proxy-Connection: Keep-Alive
 
 GET /path/10600002 HTTP/1.1
-Host: test.remote.haxx.se.1060:%HTTPPORT
+Host: test.remote.haxx.se.1060:8990
 Accept: */*
 
 </protocol>
diff --git a/tests/data/test1061 b/tests/data/test1061
index 05c3209..a1d7286 100644
--- a/tests/data/test1061
+++ b/tests/data/test1061
@@ -879,7 +879,7 @@ crypto
 HTTP proxy CONNECT auth Digest, large headers and chunked data
  </name>
  <command>
-http://test.remote.haxx.se.1061:%HTTPPORT/path/10610002 --proxy http://%HOSTIP:%HTTPPORT --proxy-user silly:person --proxy-digest --proxytunnel
+http://test.remote.haxx.se.1061:8990/path/10610002 --proxy http://%HOSTIP:%HTTPPORT --proxy-user silly:person --proxy-digest --proxytunnel
 </command>
 </client>
 
@@ -889,17 +889,17 @@ http://test.remote.haxx.se.1061:%HTTPPORT/path/10610002 --proxy http://%HOSTIP:%
 ^User-Agent: curl/.*
 </strip>
 <protocol>
-CONNECT test.remote.haxx.se.1061:%HTTPPORT HTTP/1.1
-Host: test.remote.haxx.se.1061:%HTTPPORT
+CONNECT test.remote.haxx.se.1061:8990 HTTP/1.1
+Host: test.remote.haxx.se.1061:8990
 Proxy-Connection: Keep-Alive
 
-CONNECT test.remote.haxx.se.1061:%HTTPPORT HTTP/1.1
-Host: test.remote.haxx.se.1061:%HTTPPORT
-Proxy-Authorization: Digest username="silly", realm="weirdorealm", nonce="12345", uri="test.remote.haxx.se.1061:%HTTPPORT", response="4e23449fa93224834299e7282a70472c"
+CONNECT test.remote.haxx.se.1061:8990 HTTP/1.1
+Host: test.remote.haxx.se.1061:8990
+Proxy-Authorization: Digest username="silly", realm="weirdorealm", nonce="12345", uri="test.remote.haxx.se.1061:8990", response="4e23449fa93224834299e7282a70472c"
 Proxy-Connection: Keep-Alive
 
 GET /path/10610002 HTTP/1.1
-Host: test.remote.haxx.se.1061:%HTTPPORT
+Host: test.remote.haxx.se.1061:8990
 Accept: */*
 
 </protocol>
diff --git a/tests/data/test206 b/tests/data/test206
index 3ddc1d9..902d0a6 100644
--- a/tests/data/test206
+++ b/tests/data/test206
@@ -77,7 +77,7 @@ crypto
 HTTP proxy CONNECT auth Digest
  </name>
  <command>
-http://test.remote.haxx.se.206:%HTTPPORT/path/2060002 --proxy http://%HOSTIP:%HTTPPORT --proxy-user silly:person --proxy-digest --proxytunnel
+http://test.remote.haxx.se.206:8990/path/2060002 --proxy http://%HOSTIP:%HTTPPORT --proxy-user silly:person --proxy-digest --proxytunnel
 </command>
 </client>
 
@@ -87,18 +87,18 @@ http://test.remote.haxx.se.206:%HTTPPORT/path/2060002 --proxy http://%HOSTIP:%HT
 ^User-Agent: curl/.*
 </strip>
 <protocol>
-CONNECT test.remote.haxx.se.206:%HTTPPORT HTTP/1.1
-Host: test.remote.haxx.se.206:%HTTPPORT
+CONNECT test.remote.haxx.se.206:8990 HTTP/1.1
+Host: test.remote.haxx.se.206:8990
 Proxy-Connection: Keep-Alive
 
-CONNECT test.remote.haxx.se.206:%HTTPPORT HTTP/1.1
-Host: test.remote.haxx.se.206:%HTTPPORT
-Proxy-Authorization: Digest username="silly", realm="weirdorealm", nonce="12345", uri="test.remote.haxx.se.206:%HTTPPORT", response="003e36decb4dbf6366b3ecb9b87c24ec"
+CONNECT test.remote.haxx.se.206:8990 HTTP/1.1
+Host: test.remote.haxx.se.206:8990
+Proxy-Authorization: Digest username="silly", realm="weirdorealm", nonce="12345", uri="test.remote.haxx.se.206:8990", response="003e36decb4dbf6366b3ecb9b87c24ec"
 Proxy-Connection: Keep-Alive
 
 GET /path/2060002 HTTP/1.1
 User-Agent: curl/7.12.3-CVS (i686-pc-linux-gnu) libcurl/7.12.3-CVS OpenSSL/0.9.6b zlib/1.1.4
-Host: test.remote.haxx.se.206:%HTTPPORT
+Host: test.remote.haxx.se.206:8990
 Accept: */*
 
 [DISCONNECT]
-- 
1.8.1.5

